docs/Configuring.md

```diff
@@ -506,27 +507,52 @@ When `invalidate-bad-cache-results=true` (default: `true`), Cromwell will invali
 
 Cromwell also accepts [Workflow Options](wf_options/Overview#call-caching-options) to override the cache read/write behavior.  
 
+#### Call cache strategy options
+
+* hash based options. These read the entire file. These strategies work with containers.
+    * `xxh64`. This uses the 64-bit implementation of the [xxHash](https://www.xxhash.com)
+             algorithm. This algorithm is optimized for file integrity hashing and provides a more than 10x speed improvement over
+             md5. 
+    * `md5`. The well-known md5sum algorithm
+* Path based options. These are based on filepath. Extremely lightweight, but only work with the `soft-link` file 
+caching strategy and can therefore never work with containers.
+    * `path` creates a md5 hash of the path.
+    * `path+modtime` creates a md5 hash of the path and its modification time.
+* Fingerprinting. This strategy works with containers.
+    * `fingerprint` tries to create a fingerprint for each file by taking its last modified time (milliseconds since
+       epoch in hexadecimal) + size (bytes in hexadecimal) + the xxh64 sum of the first 10 MB of the file. It is much
+       more lightweight than the hash based options while still unique enough that collisions are infeasible. This 
+       strategy works well for workflows that generate multi-gigabyte files and where hashing these files on the 
```

Comment:
Given that cache copying might move files around, the timestamps will potentially be different for files which you want to treat as the same. So you might want to mention that this works best with specific cache copying strategies?

Reply:
Good point...
This works specifically well for hard-links, which do preserve the mod time.
However for copies you are correct. Here a new hash will have to be calculated. This is a good caveat to mention.